staff/maryam-malek/cv-styled/style.css

```diff
@@ -0,0 +1,128 @@
+body {
+    background-color: grey;
+    box-sizing: border-box;
+}
+.main {
+    display: grid;
+    grid-template: 1fr 1fr 1fr / 10% 1fr 1fr 10%; 
```

Comment:
if you are going to more than one row or column with the same dimensions you can use:
repeat (3, 1fr) for example. saves you some typing!